tron/bin/tron_stateless_alert.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3.6
+import argparse
+import logging
+import os
+import time
+
+from tron.config import manager
+from tron.config import schema
+from tron.serialize.runstate.dynamodb_state_store import DynamoDBStateStore
+
+# Default values for arguments
+DEFAULT_WORKING_DIR = '/var/lib/tron/'
+DEFAULT_CONF_PATH = 'config/'
+DEFAULT_ALERT_AFTER = 30
+log = logging.getLogger('tron_stateless_alert')
+
+
+def get_last_run_time(job):
+    '''
+    Get all sorted timestamps, and only count the actions that actually ran
+    '''
+    timestamps = []
+    job_runs = job['runs']
+    for run in job_runs:
+        for action in run['runs']:
+            if action.get('start_time') and action.get('state') != 'scheduled':
+                timestamps.append(action.get('start_time'))
+    timestamps.sort()
+    return timestamps[-1] if timestamps else None
+
+
+def parse_cli():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-w",
+        "--working-dir",
+        default=DEFAULT_WORKING_DIR,
+        help="Working directory for the Tron daemon, default %(default)s",
+    )
+    parser.add_argument(
+        "-c",
+        "--config-path",
+        default=DEFAULT_CONF_PATH,
+        help="File path to the Tron configuration file",
+    )
+    parser.add_argument(
+        "--job-for-stateless-alert",
+        required=True,
+        help="The job name to read timestamp from",
+    )
+    parser.add_argument(
+        "--alert-after",
```

Comment:
Don't use `alert_after` for this.
How about `staleness_threshold` and make it in seconds?